group-coordinator/src/main/java/org/apache/kafka/coordinator/group/streams/TopologyMetadata.java

```diff
@@ -36,7 +36,7 @@
 public record TopologyMetadata(MetadataImage metadataImage, SortedMap<String, ConfiguredSubtopology> subtopologyMap) implements TopologyDescriber {
 
     public TopologyMetadata {
-        metadataImage = Objects.requireNonNull(metadataImage);
+        Objects.requireNonNull(metadataImage);
         subtopologyMap = Objects.requireNonNull(Collections.unmodifiableSortedMap(subtopologyMap));
```

Comment:
Nitpick. The same applies to line36 in `StreamsGroupHeartbeatResult.java`.
```suggestion
        subtopologyMap = Collections.unmodifiableSortedMap(Objects.requireNonNull(subtopologyMap));
```

Reply:
I'm not sure I understand your comment. Can you clarify what you mean?

Reply:
If the passed `subtopologyMap` is null
Current: Collections.unmodifiableSortedMap throws NPE (so Objects.requireNonNull never takes effect).
After: Objects.requireNonNull throws NPE.

Also, this results in a different stack trace if NPE.

Reply:
Good point, done